### DISCUSSION TITLE
Webhooks event: get conversation started

### DISCUSSION THREAD

**Discussion by jalbertsr** [chatwoot/chatwoot#2905] [Q&A]
I'm using the webhook and I want to know when a chat is opened/started. I can read that `Currently webhooks support only message_created event`. 

Is there a way to know when a chat is opened and the user started a conversation with only this event without having to store somewhere the conversation id? 

When all the other events that are available for API callback will be able via webhooks?

**Comment by sojan-official** [comment]
Could you point to where you found `Currently webhooks support only message_created event` ?  

Webhooks support additional events as well : https://www.chatwoot.com/docs/product/others/webhook-events

There is a webhook event  `webwidget_triggered` which lets you know when a visitor has started the chat. 
https://www.chatwoot.com/docs/product/others/webhook-events#webwidget_triggered

the payload will contain a conversation_id if the user has started a conversation.

**Reply by jalbertsr** [reply]
I found it here https://www.chatwoot.com/docs/product/features/webhooks in the first paragraph. 
![Screenshot 2021-09-07 at 14 27 57](https://user-images.githubusercontent.com/9289219/132344522-e6038312-fc55-462f-a522-2e8377bec87b.png)

And actually, I guess it's correct since it's the only event I'm receiving...

**Reply by jalbertsr** [reply]
So what documentation is correct? Does it has only one event or all the events on the list? 

Thanks in advance.

**Reply by sojan-official** [reply]
@jalbertsr  There shouldn't be a reason not to support the other events as well. We are currently taking a look at this

**Reply by jalbertsr** [reply]
Thanks a lot, let me know if there is any change please :)

**Comment by sojan-official** [comment]
@jalbertsr  We will be working on some improvements over the webhooks as captured here : https://github.com/chatwoot/chatwoot/issues/3140 

I can confirm at the moment the only events supported are `message_created`, `message_updated` , `conversation_status_changed` and `webwidget_triggered`

**Reply by CelestineAkpanoko** [reply]
Hi @sojan-official, I hope you are well. Please can you describe how to create a webwidget_triggered event? I am attempting to get the chatbot to send a message (start conversation) after the user clicks on the live widget.

**Reply by sojan-official** [reply]
@CelestineAkpanoko if you use agent bot APIs or have a webhook configured. Whenever a contact clicks on the chat bubble and the conversation window opens up you will receive the webhook event `webwidget_triggerd` 

just make sure to have the option checked when you are configuring webhooks.
